src/metabase/config.clj

```diff
@@ -101,6 +102,13 @@
    Looks something like `Metabase v0.25.0.RC1`."
   (str "Metabase " (mb-version-info :tag)))
 
+;; This variable used to live at `metabase.metabot.instance/local-process-uuid`
+(defonce ^{:doc "This UUID is randomly-generated upon launch and used to identify this specific Metabase instance during this specifc
+                run. Restarting the server will change this UUID, and each server in a hortizontal cluster will have its own ID,
+                making this different from the `site-uuid` Setting. The local process UUID is used to differentiate different
+                horizontally clustered MB instances so we can determine which of them will handle MetaBot duties."}
```

Comment:
Since this is not MetaBot specific anymore please update the docstring 😉

Reply:
:heavy_check_mark: